src/app/components/Footer/README.md

```diff
@@ -0,0 +1,24 @@
+# Footer
+
+[Storybook link](https://simorghstorybook.now.sh/?selectedKind=Footer)
```

Comment:
Could this be consistent with https://github.com/BBC-News/simorgh/pull/947/files#diff-b3af0b95ae347c7f3a6fc2e0dd68e2ecR1?